src/alda/sound.clj

```diff
@@ -5,10 +5,20 @@
                                      parse-position)]
             [taoensso.timbre :as    log])
   (:import [com.softsynth.shared.time TimeStamp ScheduledCommand]
-           [com.jsyn.engine SynthesisEngine]))
+           [com.jsyn.engine SynthesisEngine]
+           [javax.sound.midi Sequence]
+           [javax.sound.midi MidiSystem]
+           [javax.sound.midi ShortMessage]
+           [javax.sound.midi Synthesizer]
+           [java.io File]
+           ))
 
 (def ^:dynamic *synthesis-engine* nil)
 
+;; FIXME workers don't die properly when nil?
+;; TODO set this to true
+(def +use-jysn+ nil)
```

Comment:
I think for now, a good thing to do would be to make this a dynamic var, and flip the logic so that it's true if we're NOT using JSyn. In other words:

```clojure
(def ^:dynamic *use-midi-sequencer* false)
```

That way, we get the default behavior (use JSyn the way we do currently), but calling code can optionally enable this new feature by binding `*use-midi-sequencer*` to true:

```clojure
(binding [*use-midi-sequencer* true]
  (play! ...))
```

This will make it easier to add this as an experimental new feature without changing the default behavior.